session_items.py

```diff
@@ -36,82 +38,31 @@ def add_item(title):
     Returns:
         item: The saved item.
     """
-    items = get_items()
-
-    # Determine the ID for the item based on that of the previously added item
-    id = items[-1]['id'] + 1 if items else 0
-
-    item = { 'id': id, 'title': title, 'status': 'Not Started' }
-
-    # Add the item to the list
-    items.append(item)
-    session['items'] = items
-
-    return item
+    payload = {
+        'key': os.getenv('TRELLO_KEY'), 
+        'token': os.getenv('TRELLO_TOKEN'),
+        'idList': os.getenv('TRELLO_TODO_LIST'), 
+        'name': title
+    }
+    r = requests.post('https://api.trello.com/1/Cards', params=payload)
 
 
-def save_item(item):
-    """
-    Updates an existing item in the session. If no existing item matches the ID of the specified item, nothing is saved.
-
-    Args:
-        item: The item to save.
-    """
-    existing_items = get_items()
-    updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-
-    session['items'] = updated_items
-
-    return item
-
 def delete_todo(todo_id):
-    existing_items = get_items()
-    session['items'] = [ items for items in existing_items if int(items.get('id')) != int(todo_id) ]
-    return todo_id
-
-
-def complete_todo(id):
-  
-    existing_items = get_items()
-
-    for item in existing_items:
-        if item['id'] == int(id):
-            item['status'] = "Completed"
-            break
-
-    session['items'] = existing_items
-
-    return id
-
-
-def started_todo(id):
-  
-    existing_items = get_items()
-
-    for item in range(len(existing_items)):
-        if existing_items[item]['id'] == int(id):
-            existing_items[item]['status'] = "Started"
-            break
-
-    session['items'] = existing_items
-
-    return id
-
-
-#update and status buttons
-
-def update_status(items, status):
-    if (status.lower().strip() == 'not started'):
-        status = NOTSTARTED
-        print("Invalid Status: " + status)
-        return None
-
-def update_item(item_id, new_todo_value, new_status_value):
```

Comment:
You've removed this function, but you still have the update_todo function in app.py that tries to use it (and a form in index.html that tries to use that endpoint).

So those things should be deleted or at least update update_todo() in app.py to return "not implemented"

Same for session.update_status/update_todo in app.py